__tests__/app.test.js

```diff
@@ -118,7 +118,66 @@ describe('app', () => {
             .then((response) => {
                 const responseMessage = response.body.msg
                 expect(responseMessage).toBe('invalid input id')
+            });
+        });
+    })
+    describe('GET/api/reviews/:review_id/comments', () => {
+        test('200: GET responds with empty array if there are no comments with given review_id', () => {
+            return request(app)
+            .get('/api/reviews/1/comments')
+            .expect(200)
+            .then(({body}) => {
+                const comments = body
+                expect(comments.length).toBe(0)
+                expect(comments).toEqual([])
+            })
+        });
+        test('200: GET responds with an array of comment for given review_id', () => {
+            return request(app)
+            .get('/api/reviews/2/comments')
+            .expect(200)
+            .then(({body}) => {
+                const comments = body
+                expect(comments.length).toBe(3)
+                expect(comments).toBeSorted({descening: true})
+                comments.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        comment_id: expect.any(Number),
+                        review_id: 2,
+                        created_at: expect.any(String),
+                        votes: expect.any(Number)
+                    })
+                })
+            })
         });
     });
-})
+    describe('Errors for GET/api/reviews/:review_id/comments', () => {
+        test('400: responds with 400 and msg when incorrect id input', () => {
+            return request(app)
+            .get('/api/reviews/anything/comments')
+            .expect(400)
+            .then((response) => {
+                const responseMessage = response.body.msg
+                expect(responseMessage).toBe('invalid input id')
+            });
+        });
+    })
```

Comment:
you've got a test for invalid id but are missing one for valid but non existent e.g /api/reviews/1000/comments
I would recommend add ing it in here